app/javascript/src/components/editor/Modals/Modals.svelte

```diff
@@ -21,5 +21,5 @@
 </script>
 
 {#if component}
-  <svelte:component this={component} />
+  <svelte:component this={component} {... $modal.parameters ?? {}} />
```

Comment:
2 things:
- Spread has a space (I didn't even know that was allowed)
- I think it would make more sense to use `||` rather than `??`. If the value is falsey, it will error.
```suggestion
  <svelte:component this={component} {...$modal.parameters || {}} />
```